requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
+coverage
+flake8
```

Comment:
Again consider the structure in the operator template (but in a future PR)

Reply:
I took a note of it for the next PR.